apps/logo-quiz/src/shared/services/level.service.ts

```diff
@@ -6,8 +6,15 @@ import { authHeader } from '../helpers';
 export function fetchLevels() {
   return axios.get<Level[]>(`http://localhost:3333/api/levels`, {
     headers: authHeader()
-  })
-    .then((level: AxiosResponse<Level[]>) => {
-      return level.data;
-    });
+  }).then((levels: AxiosResponse<Level[]>) => {
+    return levels.data;
+  });
+}
+
+export function fetchLevel(id: String) {
+  return axios.get(`http://localhost:3333/api/levels/${id}`, {
```

Comment:
should we move this localhost to a configuration variable?

Reply:
same in other places

Reply:
Yeah, sounds good, I'll do it